src/qibojit/backends/__init__.py

```diff
@@ -1,2 +1,46 @@
+from qibo.config import raise_error
+
 from qibojit.backends.cpu import NumbaBackend
 from qibojit.backends.gpu import CupyBackend, CuQuantumBackend
+
+QibojitBackend = NumbaBackend | CupyBackend | CuQuantumBackend
+
+PLATFORMS = ("numba", "cupy", "cuquantum")
+
+
+class MetaBackend:
+    """Meta-backend class which takes care of loading the qibojit backends."""
+
+    @staticmethod
+    def load(platform: str) -> QibojitBackend:
+        """Loads the backend.
+
+        Args:
+            platform (str): Name of the backend to load: either `numba`, `cupy` or `cuquantum`.
+        Returns:
+            qibo.backends.abstract.Backend: The loaded backend.
+        """
+
+        if platform == "numba":
+            return NumbaBackend()
+        elif platform == "cupy":
+            return CupyBackend()
+        elif platform == "cuquantum":
+            return CuQuantumBackend()
+        else:
+            raise_error(
+                ValueError,
+                f"Unsupported platform, please use one among {PLATFORMS}.",
+            )
```

Comment:
In principle, you could even implement this with a `dict`. Or, even better, an `Enum`.

```suggestion
        return Platforms[platform].value()
```
and above, changing the constant to:
```py
class Platforms(Enum):
    numpy = NumpyBackend
    ...
```